### DISCUSSION TITLE
Success in this field

### DISCUSSION THREAD

**Discussion by chainflash** [Cyfrin/foundry-full-course-cu#3056] [General]
Hello everyone, here I would like to talk about the successes and income you have achieved in this field, or anything that could motivate other members of the community.
Please share your comments.

**Comment by EngrPips** [comment]
This seems Interesting, I am looking forward to reading all those success stories.

**Reply by chainflash** [reply]
yes cool 💯

**Comment by vigneshbalu96** [comment]
Looks like people are busy or got in the whirl of learning the things or feeling intimidated by the web3 world or less people got into the 
:rocket:  space... :thinking: 

cheers to the Blockchaineers though!

**Reply by EngrPips** [reply]
I think we will all agree that it is a tough space.

**Comment by vigneshbalu96** [comment]
@PatrickAlphaC Your insight would be greatly appreciated here. It’s widely acknowledged that many people are securing jobs or freelancing opportunities in the web3 space after completing this phenomenal course. However, some individuals are feeling intimidated while pursuing roles as fellow Blockchianeer's and blockchain enthusiasts. 

Could you shed some light 🔦 on potential solutions to tackle this situation? Your guidance would bring much-needed clarity to those in our field.

Thanks in advance! 🙏

**Reply by chainflash** [reply]
greate thanks 👍 
I also want other developers to tag Patrick on this page so that the chances of him seeing this here increase, and we can benefit from his guidance.
@PatrickAlphaC

**Reply by EngrPips** [reply]
I trust he will respond as soon as he is chanced.

**Reply by PatrickAlphaC** [reply]
Hi! Sorry, I slowed down for the new year. And sure! So here are my thoughts:

1. New things can be scary! 
2. Hackathons & Build your portfolio. If your github doesn't have a bunch of badass projects you made, that's a bad start!
3. Try contributing to open sourced projects. Find a project, and maybe try to solve some of their issues
4. For security researchers, your path is the easiest, just kill it on codehawks!

**Reply by chainflash** [reply]
Happy new year 🌟
Great, thank you for your opinion
